src/test_utils.ts

```diff
@@ -69,3 +69,27 @@ export function newCustomerWithNodeOptions(options: GoogleAdsNodeOptions) {
 export function getRandomName(entity: string) {
     return `test-${entity}-${(Math.random() * 100000 + 1).toFixed(0)}`
 }
+
+
+export async function getCustomerAccessToken() {
+
+    const data =  JSON.parse(process.env.GOOGLE_ADS_SERVICE_ACCOUNT as string);
```

Comment:
@avermeil to make this test pass we need an env in the CI with a real service account.

Do you think is a good idea?